packages/atlas/src/providers/assets/operatorsProvider.tsx

```diff
@@ -72,7 +90,19 @@ export const OperatorsContextProvider: FC<PropsWithChildren<unknown>> = ({ child
         // we need to filter operators manually as query node doesn't support filtering this deep
         const operatorsInfos: OperatorInfo[] = bucket.operators
           .filter((operator) => operator.metadata?.nodeEndpoint?.includes('http') && operator.status === 'ACTIVE')
-          .map((operator) => ({ id: operator.id, endpoint: operator.metadata?.nodeEndpoint || '' }))
+          .map((operator) => ({
+            id: operator.id,
+            endpoint: operator.metadata?.nodeEndpoint || '',
+            distance: operator.metadata?.nodeLocation?.coordinates
```

Comment:
We access `operator.metadata?.nodeLocation?.coordinates` 3 times here, can we store in some temporary variable?